engine/apps/heartbeat/models.py

```diff
@@ -82,120 +124,7 @@ def check_heartbeat_state(self):
                 state_changed = True
         return state_changed
 
-    def on_heartbeat_restored(self):
-        raise NotImplementedError
-
-    def on_heartbeat_expired(self):
-        raise NotImplementedError
-
-    @property
-    def is_expired(self):
-        return self.last_heartbeat_time + timezone.timedelta(seconds=self.timeout_seconds) < timezone.now()
-
-    @property
-    def expiration_time(self):
-        return self.last_heartbeat_time + timezone.timedelta(seconds=self.timeout_seconds)
-
-
-class HeartBeat(BaseHeartBeat):
-    """
-    HeartBeat Integration itself
-    """
-
-    alert_receive_channel = models.ForeignKey(
-        "alerts.AlertReceiveChannel", on_delete=models.CASCADE, related_name="heartbeats"
-    )
-
-    message = models.TextField(default="")
-    title = models.TextField(default="HeartBeat Title")
-    link = models.URLField(max_length=500, default=None, null=True)
-    user_defined_id = models.CharField(default="default", max_length=100)
-
-    def on_heartbeat_restored(self):
-        create_alert.apply_async(
-            kwargs={
-                "title": "[OK] " + self.title,
-                "message": self.title,
-                "image_url": None,
-                "link_to_upstream_details": self.link,
-                "alert_receive_channel_pk": self.alert_receive_channel.pk,
-                "integration_unique_data": {},
-                "raw_request_data": {
-                    "is_resolve": True,
-                    "id": self.pk,
-                    "user_defined_id": self.user_defined_id,
-                },
-            },
-        )
-
-    def on_heartbeat_expired(self):
-        create_alert.apply_async(
-            kwargs={
-                "title": "[EXPIRED] " + self.title,
-                "message": self.message
-                + "\nCreated: {}\nExpires: {}\nLast HeartBeat: {}".format(
-                    self.created_at,
-                    self.expiration_time,
-                    self.last_checkup_task_time,
-                ),
-                "image_url": None,
-                "link_to_upstream_details": self.link,
-                "alert_receive_channel_pk": self.alert_receive_channel.pk,
-                "integration_unique_data": {},
-                "raw_request_data": {
-                    "is_resolve": False,
-                    "id": self.pk,
-                    "user_defined_id": self.user_defined_id,
-                },
-            }
-        )
-
-    class Meta:
-        unique_together = (("alert_receive_channel", "user_defined_id"),)
```

Comment:
this model does not appear to be used anywhere. It's table has zero data in it in our production/staging databases. Lets remove it.